scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -24,6 +26,8 @@ public static class PublishTargets
 
         private static string SharedFrameworkNugetVersion { get; set; }
 
+        private static List<string> _latestBlobs;
```

Comment:
Peter and I talked about that offline and we ended up with the static variable being a little more clear than an out var. I'm open to re-opening that discussion if you want :)